src/plot_utils.py

```diff
@@ -321,7 +318,7 @@ def guess_column_type(df_column):
         return guess_column_type(df_column)
 
 
-def conv_categorical_vals_to_numeric(data_srs, bdb=None, generator_name=None):
+def conv_categorical_vals_to_numeric(data_srs):
     # TODO: get real valuemap from btable
```

Comment:
No, I don't. I don't especially want to remove the TODO comment even if it is related, and I think reinstating the arguments if they're relevant would be pretty easy.